egui/src/widgets/drag_value.rs

```diff
@@ -262,21 +273,41 @@ impl<'a> Widget for DragValue<'a> {
                 .sense(Sense::click_and_drag())
                 .text_style(TextStyle::Monospace)
                 .wrap(false);
-            let response = ui.add_sized(ui.spacing().interact_size, button);
-            let response = response.on_hover_text(format!(
+
+            let mut hover_text = format!(
                 "{}{}{}\nDrag to edit or click to enter a value.",
                 prefix,
                 value as f32, // Show full precision value on-hover. TODO: figure out f64 vs f32
                 suffix
-            ));
+            );
+
+            if is_slow_speed {
+                hover_text.push_str("\nPress 'Shift' while dragging for better control.");
+            }
+
+            let response = ui.add_sized(ui.spacing().interact_size, button);
+            let response = response
+                .on_hover_cursor(CursorIcon::ResizeHorizontal)
+                .on_hover_text(hover_text);
 
             if response.clicked() {
                 ui.memory().request_focus(kb_edit_id);
                 ui.memory().drag_value.edit_string = None; // Filled in next frame
             } else if response.dragged() {
+                response.ctx.output().cursor_icon = CursorIcon::ResizeHorizontal;
```

Comment:
```suggestion
                ui.output().cursor_icon = CursorIcon::ResizeHorizontal;
```

just shorter, and also more consistent with the rest of egui codebase